src/Components/Search/SearchInputContainer.tsx

```diff
@@ -25,12 +25,19 @@ const SearchButton = styled.button`
   }
 `
 
-export const SearchInputContainer: React.ExoticComponent<any> = React.forwardRef((props, ref) => {
+export const SearchInputContainer: React.ExoticComponent<
+  any
+> = React.forwardRef((props, ref) => {
   return (
     <Box style={{ position: "relative" }}>
       <Input ref={ref} style={{ width: "100%" }} {...props} />
       <SearchButton>
-        <svg width="18" height="18" xmlns="http://www.w3.org/2000/svg">
+        <svg
+          width="18"
+          height="18"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ position: "relative", top: 3 }}
```

Comment:
Is this intentional? It seems like a change to the positioning of an icon within the SearchInputContainer wouldn't be required for these changes.

Reply:
This was noticed once I removed the inner padding from the Input, which moved the icon slightly. (I'm noticing that all of the icons tend to be [a few pixels off](https://github.com/artsy/reaction/pull/2093/files#diff-349f764cbaa09fc0d16c2f1c1f033f4aR163) position-wise, which could be fixed once Jon updates palette. This could possibly be fixed by placing the [Icon base component](https://github.com/artsy/palette/blob/master/src/svgs/Icon.tsx#L29) inside of a Flex container like:

```
<Flex alignItems='center'>
  <Icon ... />
</Flex> 
```

Might need to play with it a bit though.

Reply:
Also, I see now that those changes are about to get merged into master via #2090 anyway. Seeing them in that PR makes sense to me now. 

I imagine you'll have merge conflicts after #2090 merges, but feel free to self-merge once those get cleaned up! This is awesome.